src/analysis/retail/evoker/shared/modules/normalizers/LeapingFlamesNormalizer.ts

```diff
@@ -0,0 +1,143 @@
+import SPELLS from 'common/SPELLS';
+import EventLinkNormalizer, { EventLink } from 'parser/core/EventLinkNormalizer';
+import { Options } from 'parser/core/Module';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import {
+  ApplyBuffEvent,
+  ApplyBuffStackEvent,
+  CastEvent,
+  DamageEvent,
+  EventType,
+  GetRelatedEvents,
+  HasRelatedEvent,
+  HealEvent,
+  RefreshBuffEvent,
+} from 'parser/core/Events';
+
+export const LEAPING_FLAMES_HITS = 'leapingFlamesHits';
+export const LEAPING_FLAMES_CONSUME = 'leapingFlamesConsume';
+export const ESSENCE_BURST_GENERATED = 'essenceBurstGenerated';
+export const ESSENCE_BURST_CAST_GENERATED = 'essenceBurstCastGenerated';
+export const ESSENCE_BURST_WASTED = 'essenceBurstWasted';
+
+const LEAPING_FLAMES_HIT_BUFFER = 1000;
+const ESSENCE_BURST_BUFFER = 20;
+const LEAPING_FLAMES_CONSUME_BUFFER = 35;
```

Comment:
this buffer might be a little tight, but that is easy to change later if it ever is actually a problem